v5/owner.apib

```diff
@@ -0,0 +1,449 @@
+# Group Owner
+
+## Owner [/owner]
+Access needs a __private key__.
+
+### List parking spaces [GET /owner/parking-spaces]
+Returns all parking spaces that are owned by the user given by the used private token for the request.
+
++ Request
+    + Headers
+
+            Authorization: Basic MDEyMzQ1Njc4YWJjZGVmOnBhcmt1
+
++ Response 200 (application/json)
+    + Attributes (Parking space)
+
+### Retrieve a Parking space [GET /owner/parking-spaces/{parking_space_id}]
+Retrieves the details of one singe parking space.
+
++ Parameters
+    + parking_space_id: `005c4826-4e28-11e3-a675-d43d7eece53d` (string, required) - The id or code of the parking space
++ Request
+    + Headers
+
+            Authorization: Basic MDEyMzQ1Njc4YWJjZGVmOnBhcmt1
+
++ Response 200 (application/json)
+    + Attributes (Parking space)
+
++ Response 404
+Returned if parking space was not found or it doesn't belong to the user.
+    + Body
+
+### Update a Parking space [PUT /owner/parking-spaces/{parking_space_id}]
+Updates the parking space by setting the values of the parameters passed. Any parameter not provided will be left unchanged.
+
++ Parameters
+    + parking_space_id : `00cd7cfd-e42d-11e2-8bf1-8a83f3373875` (string, required) - the parking space id
+
++ Request  (application/json)
+    + Attributes
+        + `status`: `inactive` (string)
+        + `code`: `ASDF12` (string)
+        + `type`: `offstreet` (string)
+        + `price` (Owner Price)
+        + `options` (array) - describes which options are available at the location and can contains each of the following elements. The list can be **empty**.
+            + (enum)
+              + `excess length`
+              + `indoor`
+        + `post_booking_info`: `Hallo, bitte lassen Sie mich allein.`
```

Comment:
Haha, I know this sentence was funny enough for internal documentation, but I feel we should use a different example for stuff that is visible to the public e.g. `Please leave a note with your phone number in the car.`

Reply:
It was funny but I changed it.